templates/unminified/cache.tpl.js

```diff
@@ -0,0 +1,125 @@
+(function() {
+  'use strict';
+
+  if (!('URLSearchParams' in window)) {
+    // Simple polyfill for URLSearchParams#has() and URLSearchParams#get().
+    class URLSearchParamsPolyfill {
```

Comment:
The transpiling is done as part of the minification. If Closure Compiler is used as described in the Readme, then it is also automatically transpiled to es5.